<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyApp</class>
 <widget class="QMainWindow" name="MyApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>악성코드 분석 도구</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="preprocessing_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="vir_result">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>871</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>871</width>
      <height>221</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="train_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>111</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>1341</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>661</width>
      <height>451</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_fileSelection">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_malware_file">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>악성코드 파일:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="malware_file">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>No File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="data_select">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>파일 선택</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_api_key">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_api_key">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>VirusTotal API Key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="api_key_input">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>API 키를 입력하세요</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_processingStatus">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_status_preprocessing">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_data_preprocessing">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>데이터 전처리</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_data_preprocessing">
           <property name="minimumSize">
            <size>
             <width>49</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>대기 중</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_status_training">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_model_training">
           <property name="minimumSize">
            <size>
             <width>63</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>모델 학습</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_model_training">
           <property name="minimumSize">
            <size>
             <width>62</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>대기 중</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="train_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #007bff; color: white; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>분석 시작</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
